lib/rock_rms/resources/payment_detail.rb

```diff
@@ -0,0 +1,61 @@
+module RockRMS
+  class Client
+    module PaymentDetail
+      def list_payment_details(options = {})
+        options = {'$top' => 10}
```

Comment:
This overwrites the options whenever they are passed in.

Reply:
Perhaps it would be best to either:
a) set these as the default options
b) set the `$top` parameter if it's not already set